pygenn/genn_groups.py

```diff
@@ -278,6 +300,14 @@ def delay_slots(self):
     def size(self):
         return self.pop.get_num_neurons()
 
```

Comment:
Would be cool to have a per-layer `pull_recording_buffers_from_device` method, so you can selectively pull recording buffers without pulling anything else.

Reply:
What would be the use case for doing that? If you've enabled recording on a neuron population wouldn't you always want the data at the end of the recording period?

Reply:
Good point, I suppose you would.